unitTests/utils/SearchboxResizeTest.ts

```diff
@@ -0,0 +1,87 @@
+import * as Mock from '../MockEnvironment';
+import { Searchbox } from '../../src/ui/Searchbox/Searchbox';
+import { ISearchboxOptions } from '../../src/ui/Searchbox/Searchbox';
+import { $$ } from '../../src/utils/Dom';
+
+interface HTMLElements {
```

Comment:
Everything you declared should be done inside the "describe" scope of your tests, just a bit cleaner.